test/methods.jl

```diff
@@ -108,7 +109,7 @@ end
         @test_throws ArgumentError Concave(0)
         @test_throws ArgumentError Concave(-2)
 
-        method = Absolmin(0)
+        method = Absolmin(1e-5)
```

Comment:
Why do we need to change `epsilon` here?

Reply:
Then we should disallow any `epsilon <= 0` at construction time of the criterion.